imports/ui/employee/employeeItem.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Persona } from "office-ui-fabric-react";
+
+const EmployeeItem = ({
+	firsName,
+	lastName,
+	email,
+	size, //normal/large
+}) => (
+	<div>
+		<Persona
+		className={size}
```

Comment:
I would not use `className` here to set the `size`. Also, I didn't see any associated style related to this className in the style folder. Please correct me if I miss it.

I think we can directly use props in Persona here. 
The example I found: https://github.com/OfficeDev/office-ui-fabric-react/blob/7c63d63055b38cfaef0a811ed6b06a7e573e660b/packages/office-ui-fabric-react/src/components/Persona/examples/Persona.Presence.Example.tsx#L33

Reply:
@nerocui  what do you think? since before u were setting up the size in the classname way

Reply:
The Persona size is for the profile picture. We want a configurable size for the whole thing. But good catch, we probably need to use the Persona size in the future too.